mcc/resources/schemas/dbscripts/sqlserver/mcc-20.003-20.004.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE mcc.animalRequests (
+	rowid 									int identity(1,1) 	NOT NULL,
+	objectid 								VARCHAR(40)   			NOT NULL,
```

Comment:
Writing this here to keep it all in one place. like we discussed, you need to update mcc.xml to add these new tables. there is a unit test and helper here:

Go here and run: SchemaXMLTestCase:
http://localhost:8080/labkey/junit/begin.view?

This is a direct link to that test:
http://localhost:8080/labkey/junit/run.view?testCase=org.labkey.core.admin.test.SchemaXMLTestCase

It should fail for MCC module. It will give you a link to generate a template schema XML file. You probably want to start with this, but update some properties, like isHidden, columnTitle, and nullable. You might also make a FK between coinvestigators.requestid and animalRequests.objectid.  here is one example:

```
            <column columnName="vcfId">
                <columnTitle>VCF File</columnTitle>
                <nullable>true</nullable>
                <fk>
                    <fkDbSchema>sequenceanalysis</fkDbSchema>
                    <fkTable>outputfiles</fkTable>
                    <fkColumnName>rowid</fkColumnName>
                </fk>
            </column>
```


https://github.com/bimberlabinternal/BimberLabKeyModules/blob/discvr-21.3/mGAP/resources/schemas/mgap.xml